build-support/ci_lists/integration_chroot_blacklist.txt

```diff
@@ -1,4 +1,5 @@
 tests/python/pants_test/backend/jvm/tasks/jvm_compile/java:cache_compile_integration
+tests/python/pants_test/backend/jvm/tasks/jvm_compile/rsc:rsc_compile_integration_not_remoted
```

Comment:
Why is this necessary? Remoting should work fine here, since this isn't actually causing the pants runs in the integration test to be remoted.

Reply:
still under investigation: https://pantsbuild.slack.com/archives/C15HUUSL9/p1570830273030900?thread_ts=1570748859.014500&cid=C15HUUSL9